src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -433,18 +406,4 @@ protected function denormalizeParameter(\ReflectionClass $class, \ReflectionPara
 
         return $parameterData;
     }
-
-    /**
-     * @internal
-     */
-    protected function createChildContext(array $parentContext, string $attribute, ?string $format): array
-    {
-        if (isset($parentContext[self::ATTRIBUTES][$attribute])) {
-            $parentContext[self::ATTRIBUTES] = $parentContext[self::ATTRIBUTES][$attribute];
-        } else {
-            unset($parentContext[self::ATTRIBUTES]);
-        }
-
-        return $parentContext;
-    }
```

Comment:
Replaced by `CreateChildContext` trait, since that function was `@internal` I did not made a compatibility layer here ~

Reply:
Yes you're right it's allowed by the `@internal` tag